charts/fint-altinn-ebevis-service/templates/deployment.yaml

```diff
@@ -28,10 +28,10 @@ spec:
               mountPath: {{ .Values.deployment.volumeMounts.mountPath}}
           resources:
             limits:
-              memory: {{ .Values.deployment.resources.limits.memory}}
+              memory: "{{ include "fint-altinn-ebevis-service.xmx" . }}Mi"
               cpu: {{ .Values.deployment.resources.limits.cpu}}
-            requests:
-              memory: {{ .Values.deployment.resources.requests.memory}}
+            requests: 
+              memory: "{{ div (include "fint-altinn-ebevis-service.xmx" . ) 2 }}Mi"
```

Comment:
change from "{{ div (include "fint-altinn-ebevis-service.xmx" . ) 2 }}Mi" to {{ div (include "fint-altinn-ebevis-service.xmx" . ) 2 }}Mi